.github/workflows/jit.yml

```diff
@@ -134,7 +133,7 @@ jobs:
           brew install llvm@${{ matrix.llvm }}
           SDKROOT="$(xcrun --show-sdk-path)" \
             ./configure --enable-experimental-jit ${{ matrix.debug && '--with-pydebug' || '--enable-optimizations --with-lto' }}
-          make all --jobs 4
+          SDKROOT="$(xcrun --show-sdk-path)" make all --jobs 4
```

Comment:
Since we're doing this on line 134 too, maybe just remove both and put `export SDKROOT="$(xcrun --show-sdk-path)"` before the `./configure` step?

Reply:
Thanks!
Done